sessionctx/binloginfo/binloginfo_test.go

```diff
@@ -122,11 +128,15 @@ func (s *testBinlogSuite) SetUpSuite(c *C) {
 }
 
 func (s *testBinlogSuite) TearDownSuite(c *C) {
-	s.ddl.Stop()
+	err := s.ddl.Stop()
+	c.Assert(err, IsNil)
 	s.serv.Stop()
-	os.Remove(s.unixFile)
+	// to fix: file not found
+	err = os.Remove(s.unixFile)
+	c.Assert(err, NotNil)
```

Comment:
Assert that the err is either nil or "file not found".

Reply:
Added error message check.

Reply:
No, I mean, the error might be nil. E.g. when the file was not deleted successfully since the last run.
So the assert should be:

```
if err != nil {
    c.Assert(err, ErrorMatches, fmt.Sprintf("remove %v: no such file or directory", s.unixFile))
}
```

Reply:
I see, asserts updated.